### PR TITLE
improved aedtlib_personalib_install.py to support run as standalone file or as a method of pyaedt

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: pyaedt/misc/aedtlib_personalib_install.py [hunk 1]
```diff
@@ -12,27 +12,15 @@
...
-sys_lib = True
-if len(sys.argv) == 3:
-    sys_lib = True if sys.argv[2] == "1" else False
+
 
 pid = 0
 
 
-def main():
+def add_pyaedt_to_aedt(aedt_version, is_student_version=True, use_sys_lib=False):
```
Comment (MaxJPRey) [inline pyaedt/misc/aedtlib_personalib_install.py:20]: Maybe the default value for the `is_student_version` parameter should be `False` as most users are not using student version.
```suggestion
def add_pyaedt_to_aedt(aedt_version, is_student_version=False, use_sys_lib=False):
```

File: pyaedt/misc/console_setup [hunk 2]
```diff
@@ -10,7 +10,15 @@ provided by the launcher
 """
 import sys
 import atexit
-from pyaedt import *
+try:
+   from pyaedt import *
+except ImportError:
+   import os
```
Comment (MaxJPRey) [inline pyaedt/misc/console_setup:16]: See my comment above.
```suggestion
```

**Review by Samuelopez-ansys — COMMENTED** [review]
Why the file is called "aedtlib_....py"? Should it be pyaedt_personallib_install?

File: pyaedt/misc/console_setup [hunk 3]
```diff
@@ -8,9 +8,17 @@ cpython_console.py should be run instead of this script.
...
 """
+import os
 import sys
 import atexit
-from pyaedt import *
+try:
+   from pyaedt import *
+except ImportError:
+   # debug only
+   console_path = os.path.dirname(__file__)
```
Comment (kbhagat-ansys) [inline pyaedt/misc/console_setup:18]: Suggestion to change variable name:
```suggestion
   console_setup_dir = os.path.dirname(__file__)
```
@maxcapodi78

File: pyaedt/misc/console_setup [hunk 4]
```diff
@@ -8,9 +8,17 @@ cpython_console.py should be run instead of this script.
...
+   if "PersonalLib" in console_path:
+      sys.path.append(os.path.join(console_path, "..", "..", ".."))
```
Comment (kbhagat-ansys) [inline pyaedt/misc/console_setup:20]: @maxcapodi78  I think a comment here would be helpful on what we are trying to achieve - and what are we trying to apped to path if the file is in PersonalLib

**Comment by kbhagat-ansys** [comment]
@maxcapodi78  Thoughts on renaming `console_setup` to `console_setup.py`  this way PyCharm and other IDEs can correct the format and do Pep8 parsing for us.

File: pyaedt/misc/console_setup (hunk 4 above)
Comment (maxcapodi78) [reply pyaedt/misc/console_setup]: added

**Comment by maxcapodi78** [comment]
> @maxcapodi78 Thoughts on renaming `console_setup` to `console_setup.py` this way PyCharm and other IDEs can correct the format and do Pep8 parsing for us.

I've done it. Thanks

**Review by kbhagat-ansys — APPROVED** [review]
LGTM.

**Comment by kbhagat-ansys** [comment]
@maxcapodi78  I have a branch called `toolkit_copy` where I improve the paths of the Lib files required for the Toolkits and I correct the description of the `Jupter.py_build` file. 
I was planning on creating a new PR for it, or you can include it in your changes.

**Pull request merged**